ophyd/areadetector/cam.py

```diff
@@ -106,7 +106,12 @@ class CamBase(ADBase):
     gain = ADCpt(SignalWithRBV, "Gain")
 
     image_mode = ADCpt(SignalWithRBV, "ImageMode")
+    
     manufacturer = ADCpt(EpicsSignalRO, "Manufacturer_RBV")
+    model = ADCpt(EpicsSignalRO, "Model_RBV")
+    serial_number = ADCpt(EpicsSignalRO, "SerialNumber_RBV")
```

Comment:
I believe these came in from AreaDetector R2-6: https://klauer.github.io/ad_compare/ (search for `SerialNumber_RBV` and check the first column that doesn't say '`no`'). 

It's (probably) fine if ophyd starts assuming a newer version of AreaDetector than the ancient R1-9-1, but it should at least be a very prominent part of the release notes.

Reply:
I wonder if we could read the Signal for `ADCoreVersion_RBV` and then conditionally include functionality based on that? Though I agree that assuming newer than R1-9-1 going forward is probably OK